termlink/loinc.py

```diff
@@ -93,15 +107,22 @@ def __init__(self, uri, versioned):
 
     def get_relationships(self):
         "Parses a list of `Relationship` objects."
-        path = os.path.join(self.uri.path, 'MultiAxialHierarchy.csv')
-        hierarchy = etl.fromcsv(path, delimiter=',')
-        hierarchy = etl.cut(hierarchy, ['IMMEDIATE_PARENT', 'CODE', 'CODE_TEXT'])
+        core_file = _find_loinc_table_core_file(self.uri.path)
+        core = etl.fromcsv(core_file, delimiter=',')
+        core = etl.cut(core, ['LOINC_NUM', 'LONG_COMMON_NAME'])
+        hierarchy_file = _find_multi_axial_hierarchy_file(self.uri.path)
+        hierarchy = etl.fromcsv(hierarchy_file, delimiter=',')
+        hierarchy = etl.leftjoin(hierarchy, core, lkey='CODE', rkey='LOINC_NUM')
+        hierarchy = etl.cut(hierarchy, ['IMMEDIATE_PARENT', 'CODE', 'CODE_TEXT', 'LONG_COMMON_NAME'])
+        hierarchy = etl.fillright(hierarchy)
```

Comment:
The LONG_COMMON_NAME isn't available for some of the concepts in the Multiaxial Hierarchy file. This line performs the following: `If LONG_COMMON_NAME is None then LONG_COMMON_NAME = CODE_TEXT`